CODE_OF_CONDUCT.md

```diff
@@ -10,30 +8,43 @@ We pledge to act and interact in ways that contribute to an open, welcoming, div
-* Private harassment is also unacceptable. No matter who you are, if you feel you have been or are being harassed or made uncomfortable by a community member, please contact the maintainers immediately. Whether you’re a regular contributor or a newcomer, we care about making this community a safe place for you and we’ve got your back.
-* Do not make casual mention of slavery or indentured servitude and/or false comparisons of one's occupation or situation to slavery. Please consider using or asking about alternate terminology when referring to such metaphors in technology.
-* Likewise any spamming, trolling, flaming, baiting or other attention-stealing behavior is not welcome.
+Respect that people have differences of opinion and that every design or implementation choice carries a trade-off and numerous costs. There is seldom a right answer.
+Bevy welcomes experimentation and iteration: if you have an idea for how something could be done better, you want to experiment with, make a PR or write an RFC to show us how it compares in practice!
 
 ## Moderation
 
 These are the policies for upholding [our community’s standards of conduct](#our-standards). If you feel that a thread needs moderation, please contact the maintainers.
-
-1. Remarks that violate the community standards of conduct, including hateful, hurtful, oppressive, or exclusionary remarks, are not allowed. (Cursing is allowed, but never targeting another user, and never in a hateful manner).
-2. Remarks that maintainers find inappropriate, whether listed in the code of conduct or not, are also not allowed.
-3. Maintainers will first respond to such remarks with a warning.
-4. If the warning is unheeded, the user will be “kicked,” i.e., kicked out of the communication channel to cool off.
-5. If the user comes back and continues to make trouble, they will be banned, i.e., indefinitely excluded.
-6. Maintainers may choose at their discretion to un-ban the user if it was a first offense and they offer the offended party a genuine apology.
-7. If a maintainer bans someone and you think it was unjustified, please take it up with that maintainer, or with a different maintainer, in private. Complaints about bans in-channel are not allowed.
-8. Maintainers are held to a higher standard than other community members. If a maintainer creates an inappropriate situation, they should expect less leeway than others.
-
 The enforcement policies in the code of conduct apply to all official venues, including Discord channels, GitHub repositories, the Twitter community and all other forums.
```

Comment:
I know this wasn't changed, but enforcement policies are really only enforceable where Bevy maintainers and org members control the forum of discussion (i.e. GitHub and Discord). On a public forum like Twitter, it seems impossible to enforce these CoC policies. Perhaps we should reword this.

Reply:
Hmm. What about things like replies to Bevy threads?

Reply:
Sure, but that's rather limited in scope. Most of this CoC is focused on person-to-person interactions within the community, and wouldn't this broadly cover any Bevy related discussions on Twitter as well? We could bar said offender from the GitHub/Discord, but that likely does nothing to stop them from engaging in toxic behavior there. 

I think this should be reworded to:

```
These policies will be enforced in all official Bevy forums (Discord, GitHub, Reddit, etc.), but rules apply broadly to anywhere members of the community interact, including but not limited to Discord, GitHub, Reddit, Twitter, HackerNews, or in person meetings.
```

Reply:
```suggestion
These policies will be enforced in all official Bevy forums (Discord, GitHub, Reddit, etc.), but rules apply broadly to anywhere members of the community interact, including but not limited to unofficial discussions of Bevy, Bevy ecosystem spaces, or in person meetings.
```